src/Controller/AbsenceController.php

```diff
@@ -277,6 +490,37 @@ public function edit(Request $request)
         $this->templateParams(array('documents' => $this->getDocuments($a->id)));
         return $this->output('absences/edit.html.twig');
     }
+    /**
+     *@Route("/absence", name="absence.save", methods = {"POST"})
+     */
+    public function save_absence(Request $request) {
+
+        // Save absence(s).
+        $this->dbprefix = $GLOBALS['dbprefix'];
+        $this->droits = $GLOBALS['droits'];
+        $this->setAdminPermissions();
```

Comment:
This 3 lines are useless.

1) nerver used in the function,
2) The same
3) variables $this->admin or $this->adminN2 never used

Reply:
Indeed, $this->admin is used as a parameter line 489. Without these lines, nothing could be saved.